python/hail/methods/tests.py

```diff
@@ -30,6 +30,56 @@ def get_dataset(self):
             Tests._dataset = hc.import_vcf('src/test/resources/sample.vcf').to_hail1().split_multi_hts().to_hail2()
         return Tests._dataset
 
+    def test_ibd(self):
+        dataset = self.get_dataset()
+
+        def plinkify(ds, min=None, max=None):
+            vcf = utils.new_temp_file(prefix="plink", suffix="vcf")
+            plinkpath = utils.new_temp_file(prefix="plink")
+            ds.to_hail1().export_vcf(vcf)
+            threshold_string = "{} {}".format("--min {}".format(min) if min else "",
+                                              "--max {}".format(max) if max else "")
+
+            plink_command = "plink --double-id --allow-extra-chr --vcf {} --genome full --out {} {}"\
+                .format(utils.get_URI(vcf),
+                                                 utils.get_URI(plinkpath),
+                                                 threshold_string)
+            result_file = utils.get_URI(plinkpath + ".genome")
+
+            syscall(plink_command, shell=True)
```

Comment:
do we want syscall or os.system? Not sure what the difference is between the two.

Reply:
syscall is subprocess.call (I imported it as `syscall` because importing it as `call` seemed maybe too generic but I didn't necessarily want to import the entire subprocess package). I've used this for basically all shell-level commands in the past, but maybe there's a betterthing?